task-manager/src/app/app.component.scss

```diff
@@ -0,0 +1,6 @@
+#task-div {
```

Comment:
it is better to call it more abstract like task-container or task-form rather than binding it to html element name